ch-covidcertificate-backend-delivery/ch-covidcertificate-backend-delivery-ws/src/main/java/ch/admin/bag/covidcertificate/backend/delivery/ws/config/WsCloudBaseConfig.java

```diff
@@ -38,27 +35,13 @@ public abstract class WsCloudBaseConfig extends WsBaseConfig {
     @Value("${push.ios.topic}")
     private String iosPushTopic;
 
-    @Bean
-    @Override
-    public DataSource dataSource() {
-        PoolConfig poolConfig =
-                new PoolConfig(dataSourceMaximumPoolSize, dataSourceConnectionTimeout);
```

Comment:
`dataSourceMaximumPoolSize`, `dataSourceConnectionTimeout` and `dataSourceLeakDetectionThreshold` are not being applied anymore. is that ok? if yes, delete these unused fields from this config

Reply:
They should be picked up by cfenv automatically from properties
https://spring.io/blog/2019/02/15/introducing-java-cfenv-a-new-library-for-accessing-cloud-foundry-services

But yes, the variables can go